### ISSUE TITLE
Basically the hour of time played is wrong...

### ISSUE THREAD

**Issue by TheCiROMG** [Puharesource/TitleManager#296]
**Describe the bug**
Basically the hour of time played is wrong...
![image](https://user-images.githubusercontent.com/61605643/91664985-fef43f00-eac8-11ea-8b98-9ee43f4cee81.png)

**To Reproduce**
1. Go to 'config.yml'
2. Click on 'the line 201'
3. Add in the line '%statistic_time_played%'
4. Play for 3 days and... 
![image](https://user-images.githubusercontent.com/61605643/91665044-5abec800-eac9-11ea-87d0-70f3903eef2e.png)

**Expected behavior**
Basically the hour of time played is wrong...

**Screenshots**
![image](https://user-images.githubusercontent.com/61605643/91665060-67432080-eac9-11ea-9a3e-3ad3e36103e5.png)

**Stacktrace / Error**
No logs error, take the placeholder "correctly"

**Server information:**
 - TitleManager version: Lastest in spigot
 - Spigot version: Paper-164
